### DISCUSSION TITLE
Emojis/Sticker wrapper under input

### DISCUSSION THREAD

**Discussion by JohannDesobry** [flyerhq/flutter_chat_ui#471] [Ideas & Requests]
It would be great if we could have the possibility to add a wrapper taking the height of the keyboard similar to this :

![IMG_A61E8D35CBA5-1](https://github.com/flyerhq/flutter_chat_ui/assets/17253756/4a6cd3ba-fef7-4611-ad9f-f22ddb9d0232)

So that each user of the plugin can implement their own custom stickers/emojis/gif inside this wrapper.

**Comment by demchenkoalex** [comment]
oof don't know about that... how would you do it?

**Reply by leeyisoft** [reply]
I relied on emoji_picker_flutter: ^2.0.0 to do a similar effect

on line 288  https://github.com/imboy-pub/imboy-flutter/blob/main/lib/page/chat/widget/chat_input.dart

EmojiPicker

![image](https://github.com/flyerhq/flutter_chat_ui/assets/6138668/fb19ec34-c058-4a06-87b4-6de4b5e7fe4d)

**Reply by demchenkoalex** [reply]
nice thanks! might be a good addition to the library

**Comment by lucasjinreal** [comment]
<img width="439" alt="image" src="https://github.com/user-attachments/assets/92e13e6a-03f0-4924-b31c-a841d36040eb">

<img width="392" alt="image" src="https://github.com/user-attachments/assets/57c34ae9-b395-440d-9998-720753a70f68">

this works just fine.
